types/block_test.go

```diff
@@ -247,7 +247,7 @@ func makeBlockID(hash []byte, partSetSize uint32, partSetHash []byte) BlockID {
 
 var nilBytes []byte
 
-// This follows RFC-6962, i.e. `echo -n '' | sha256sum`
+// This follows RFC-6962, i.e. `echo -n ” | sha256sum`
```

Comment:
This is an error in Tendermint, but an SHA-256 hash of `"` will not have this value. I'll leave it to you to fix it.